### ISSUE TITLE
RFE: packagize the metaconfiguration script

### ISSUE THREAD

**Issue by mgedmin** [zopefoundation/meta#97]
Every now and then I want to run the script to update a package, and then I realize that I can't just cd zope.sendmail && run something, I have to juggle two working directories somehow, and then I give up.

I'd like to be able to `pipx install -e ~/src/zopefoundation/meta` and have a runnable `zope-meta` (better name suggestions welcome) script in my PATH.

**Comment by mgedmin** [comment]
TBH a simple wrapper script I could put in ~/bin/ would suffice for me, as long as I can cd ~/src/zopefoundation/something and run zope-meta --help

**Comment by mgedmin** [comment]
So I tried a wrapper like this:

```
#!/bin/sh
# Workaround for https://github.com/zopefoundation/meta/issues/97
config_base=$HOME/src/zopefoundation/meta/config
if ! test -x "$config_base"/bin/python; then
    (cd "$config_base" && python3 -m venv .)
    "$config_base"/bin/pip install -r "$config_base"/requirements.txt
fi
"$config_base"/bin/python "$config_base"/config-package.py "$@"
```

and running `cd ~/src/zopefoundation/zope.interface && zope-meta .` makes config-package.py think the package name is an empty string, with sad results.

**Comment by icemac** [comment]
This will be fixed by #282.
